### ISSUE TITLE
process.browser is deprecated and browser.utils.ts is duplicated

### ISSUE THREAD

**Issue by oliveiraantoniocc** [calcom/cal.com#5360]
`process.browser` is deprecated
There are two references in the repo.
<img width="349" alt="image" src="https://user-images.githubusercontent.com/411161/199853249-755a303e-0602-45c5-9dda-82102c350464.png">

https://github.com/calcom/cal.com/blob/436f88a48e0de81a0bcbd33a32c113bd093904d5/packages/lib/browser/browser.utils.ts#L10

```ts
  if (typeof window === "undefined") {
```

it seems to be a duplication between browser.info.ts
<img width="745" alt="image" src="https://user-images.githubusercontent.com/411161/199852671-df9114ef-ac8d-425d-bf90-01ee2e18b2dc.png">

[packages/lib/browser/browser.utils.ts](https://github.com/calcom/cal.com/blob/179a9b7139f2a1117bab896a3330a220fae405fb/packages/lib/browser/browser.utils.ts)
[apps/web/lib/core/browser/browser.utils.ts](https://github.com/calcom/cal.com/blob/855aa56e7766e37773c07e8e77924fe144c171f5/apps/web/lib/core/browser/browser.utils.ts)

**Comment by MuriloAmarals** [comment]
Hey @oliveiraantoniocc Which do you think should be deleted?

**Comment by oliveiraantoniocc** [comment]
Good question, depends on how you guys are seeing this growing. 
Considering the mono-repo nature, all generic utility functions should be shared accounts in more than one web app, that's how I used to work in huge repos +180 projects. In this sense, the curse of action must be cleaning up all folders under: `apps/web/lib/core` and moving towards `packages/lib`.
In near future, if the plans are to ignite new web app solutions they would be greatly leveraged by a strong ready-to-use set of libs. Further splitting would be even better, as non-business-specific functionalities, non-UI-platform-specific (HTML or Native),... 

TL;DR
[apps/web/lib/core/browser/browser.utils.ts](https://github.com/calcom/cal.com/blob/855aa56e7766e37773c07e8e77924fe144c171f5/apps/web/lib/core/browser/browser.utils.ts)

* There are more duplications around the code, which would require more spend time, and maybe not be worth tackling right according to given available development team capacity resources x more impactful business backlog priorities, understanding this is a technical debt and may slow down developers specially newcomers can be important.

**Comment by emrysal** [comment]
Hi @oliveiraantoniocc - addressed in the attached PR; decided to move the utility function into @calcom/lib & keep HeadSeo in the @calcom/web app as this is very specific to that.
